disnake/channel.py

```diff
@@ -1406,6 +1406,7 @@ async def create_instance(
         *,
         topic: str,
         privacy_level: StagePrivacyLevel = MISSING,
+        notify: bool = False,
```

Comment:
Perhaps this should be named `notify_everyone` to make it really clear this will send a push notification to *everyone*.

Reply:
https://github.com/DisnakeDev/disnake/pull/440/commits/25fad882b53f6e815a54913d2835102025356671